src/main/io/gps.c

```diff
@@ -136,24 +136,34 @@ typedef enum {
     CLASS_NAV = 0x01,
     CLASS_ACK = 0x05,
     CLASS_CFG = 0x06,
+    CLASS_MON = 0x0a,
+    CLASS_NMEA_STD = 0xf0,
     MSG_ACK_NACK = 0x00,
     MSG_ACK_ACK = 0x01,
-    MSG_POSLLH = 0x02,
-    MSG_STATUS = 0x03,
-    MSG_DOP = 0x04,
-    MSG_SOL = 0x06,
-    MSG_PVT = 0x07,
-    MSG_VELNED = 0x12,
-    MSG_SVINFO = 0x30,
-    MSG_SAT = 0x35,
+    MSG_NAV_POSLLH = 0x02,
+    MSG_NAV_STATUS = 0x03,
+    MSG_NAV_DOP = 0x04,
+    MSG_NAV_SOL = 0x06,//
```

Comment:
```suggestion
    MSG_NAV_SOL = 0x06,
```

or add actual comment